e2e-tests/functions

```diff
@@ -364,6 +364,7 @@ compare_kubectl() {
         | yq d - '**.(name==PBM_AGENT_SERVER_ADDRESS)' \
         | yq d - 'spec.volumeClaimTemplates.*.apiVersion' \
         | yq d - 'spec.volumeClaimTemplates.*.kind' \
+        | $sed "s/${namespace}/NAME_SPACE/g" \
```

Comment:
@hors, @delgod 
It will conflict with approach which was used here
https://github.com/percona/percona-server-mongodb-operator/pull/479/commits/6ace75bd06b55b576550d44b723803e2c8feb56b
We should choose one of them.

Reply:
up to @delgod